src/Condition.php

```diff
@@ -168,6 +168,10 @@ public static function isNotNull($value)
      */
     public static function startWith($value, $comparable)
     {
+        if (is_array($comparable) || is_array($value) || is_object($comparable) || is_object($value)) {
+            return false;
+        }
+        
         if (preg_match("/^$comparable/", $value)) {
             return true;
         }
```

Comment:
In the ```startWith``` method, it lacks the default return value if the one of condition are not triggered .